src/dpos/store/index.ts

```diff
@@ -222,6 +232,7 @@ export async function refreshContractState(context: ActionContext) {
 
   state.maxYearlyRewards = fromBN(cs.maxYearlyRewards)
   state.totalWeightedStakes = fromBN(cs.totalWeightedAmountStaked)
+  state.minCandidateFee = cs.minCandidateFee
```

Comment:
`getStateAsync()` does not currently return the minimum candidate fee https://github.com/loomnetwork/loom-js/blob/8e536f39853cf3fdd6eb9be3cd893b811efcf055/src/contracts/dpos3.ts#L368

I can't find a loom-js PR that updates `getStateAsync()`.